src/unistd.rs

```diff
@@ -2879,16 +2931,22 @@ mod getres {
     /// Real, effective and saved user IDs.
     #[derive(Debug, Copy, Clone, Eq, PartialEq)]
     pub struct ResUid {
+        /// Real UID
         pub real: Uid,
+        /// effective UID
```

Comment:
Nit: I guess you forgot to capitalize this one:

```suggestion
        /// Effective UID
```